pages/_app.js

```diff
@@ -1,15 +1,31 @@
 import Layout from "../components/Layout";
 import GlobalStyles from "../styles/GlobalStyles";
 
+import {useRouter} from "next/router";
+
 function MyApp({Component, pageProps}) {
-  return (
-    <>
-      <GlobalStyles />
-      <Layout>
+  const router = useRouter();
+  const {asPath} = router;
+  console.log(asPath);
```

Comment:
Please delete this console log. There should be no console log in your final PR.

Reply:
done!